README.md

```diff
@@ -1,55 +1,21 @@
 # Museum landing page
-Implement landing page according to [Figma design](https://www.figma.com/file/cRBCqE06cDrY3s4jX7h3iY/%D0%9D%D0%90%D0%9C%D0%A3-(Edit)?node-id=0%3A1) - Use BEM and SCSS
 
-Check font styles. Use [IBM Plex Sans](https://fonts.google.com/specimen/IBM+Plex+Sans?query=ibm), [Montserrat](https://fonts.google.com/specimen/Montserrat?query=mon)
+- [DESIGN](https://www.figma.com/file/cRBCqE06cDrY3s4jX7h3iY/%D0%9D%D0%90%D0%9C%D0%A3-(Edit)?node-id=0%3A1)
+- [DEMO LINK] (https://pdabizha.github.io/Museum/)
```

Comment:
Add demo link to the PR description 
<img width="969" alt="image" src="https://github.com/user-attachments/assets/33b16b95-a37d-4dc8-8a6f-39d75e727a1b">